base/show.jl

```diff
@@ -429,6 +429,9 @@ function show_circular(io::IOContext, @nospecialize(x))
     return false
 end
 
+AnnotatedIOBuffer(io::IOContext{IOBuffer}) = AnnotatedIOBuffer(io.io, io.dict)
+IOContext(io::AnnotatedIOBuffer) = IOContext(io.io, io.dict)
```

Comment:
I don't see how these are supposed to work?

Reply:
Those are supposed to be gone in response to https://github.com/JuliaLang/julia/pull/51807#discussion_r1372536999. I'll correct this oversight shortly.